doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -824,6 +824,21 @@ Thread
 
 The issues in this section are related to the :ref:`ug_thread` protocol.
 
+.. rst-class:: v2-7-0
+
+KRKNWK-19376: OpenThread CLI sample does not work on nRF52840 dongle.
+  The sample should not use Partition manager.
+  In the main function there should be check whether the USB is already enabled to avoid a failure.
+
+  **Affected platforms:** nRF52840 Dongle
+
+  **Workaround:** While building the OpenThread CLI sample for ``nrf52840dongle/nrf52840`` target set the :kconfig:option:`SB_CONFIG_PARTITION_MANAGER` sysbuild kconfig option to ``n``.
```

Comment:
What about just instructing to cherry-pick this commit as a simpler workaround?

Reply:
Good point, but in this case, I need to add a known issue entry as a follow-up to this PR to obtain SHA. @wiba-nordic what do you think?

Reply:
I'd say using the cherry-pick is better as it is a cleaner process for the customer, even if it does mean that we need two different PRs.

Reply:
Ok, so I'm removing it from here and will create a follow-up PR with a cherry-pick :)